client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -240,8 +239,8 @@ const Checkout = React.createClass( {
 				? `/plans/${ selectedSiteSlug }/thank-you`
 				: '/checkout/thank-you/plans';
 		} else if ( selectedSite.options.is_domain_only && cartItems.hasDomainRegistration( cart ) && ! cartItems.hasPlan( cart ) ) {
-			const domainItem = find( cartItems.getAll( cart ), isDomainRegistration );
-			return domainManagementEdit( selectedSite.slug, domainItem.meta );
+			// TODO: Use purchased domain name once it is possible to set it as a primary domain when site is created.
```

Comment:
I think we should remove this comment. First because todos tend to stay forever in the code. Second because I'm not sure this will ever be possible since the process of setting a new domain as primary address is asynchronous.